person.rb

```diff
@@ -1,13 +1,14 @@
 require_relative 'nameable'
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
     super()
```

Comment:
- [ ] Because a `rental` belongs to a `book` and also belongs to a `person` we should have a `rental` method here as well and remember the `person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
resolved